Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,19 @@
-import React from 'react';
-import './Card.css';
+import React from "react";
+import "./Card.css";
+
+const CardContent = () => {
+  return (
+    <div class="card-content">
```

Comment:
Since class is a reserved keyword in javascript, you have to use `className` instead. (Your console is probably giving you an error message telling you this). Also the className needs to be written in camelCase, so `cardContent` instead, because we're messing in javascript it sees hyphens as subtraction operators.